.travis.yml

```diff
@@ -26,7 +26,7 @@ deploy:
       on:
           branch: develop
     - provider: script
-      script: 'npx semantic-release'
+      script: 'npx semantic-release && git checkout develop && git merge origin/master && git push origin develop'
```

Comment:
@JGiter Interesting. Why does this work? Does `semantic-release` apply its changes to the `master` branch (so then we need to manually copy them to the `develop` branch)?

Reply:
@jrhender I've added description